File: 0242-valid-anagram/0242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        if (s.Length != t.Length) return false;
        string sortedS = new string(s.OrderBy(c => c).ToArray());
        string sortedT = new string(t.OrderBy(c => c).ToArray());

        return sortedS == sortedT;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function(s, t) {
    let sortedS=s.split("").sort().join("");
    let sortedt=t.split("").sort().join("");
    return sortedS===sortedt
};